server/lib/flow-event.js

```diff
@@ -88,6 +94,21 @@ module.exports = (req, metrics, requestReceivedTime) => {
 
     logFlowEvent(event, metrics, req);
   });
+
+  const navigationTiming = metrics.navigationTiming;
+  if (emitPerformanceEvents) {
+    PERFORMANCE_PROPERTIES.forEach(key => {
+      const timing = navigationTiming[key];
+      if (timing > 0) {
+        const offset = timing - navigationTiming.navigationStart;
```

Comment:
One thing to watch out for, I believe these values can be `null` or `undefined`. I have performance timing disabled to give sites one less way to fingerprint me.